x/authz/keeper/keeper.go

```diff
@@ -68,6 +69,65 @@ func (k Keeper) getGrant(ctx context.Context, skey []byte) (grant authz.Grant, f
 	return grant, true
 }
 
+func (k Keeper) SetAuthzRulesKeys(ctx context.Context, options json.RawMessage) error {
+	store := k.storeService.OpenKVStore(ctx)
+
+	optionsBytes, err := json.Marshal(options)
+	if err != nil {
+		return err
+	}
+
+	authzRuleKeys := authz.AuthzRuleKeys{
```

Comment:
We should formally construct these rules. Or define a type instead of bytes, so we can marshal, unmarshal easily.